src/openzaak/conf/includes/base.py

```diff
@@ -536,6 +536,20 @@
     help_text="**EXPERIMENTAL**: indicates whether or not cloud events should be sent to the configured endpoint for specific operations on Zaak (not ready for use in production)",
 )
 
+CLOUDEVENT_SPECVERSION = config(
+    "CLOUDEVENT_SPECVERSION",
+    default="1.0",
```

Comment:
I think we can make this a fixed setting for now, if there is to be a new version, we would probably have to make code changes anyway to support it